api/v1/region_types.go

```diff
@@ -103,6 +103,70 @@ type QdrantCloudRegionStatus struct {
 	// Monitoring specifies monitoring status
 	// +optional
 	Monitoring Monitoring `json:"monitoring,omitempty"`
+	// StorageClasses contains the availble StorageClasses in the Kubernetes cluster
+	// +optional
+	StorageClasses []StorageClass `json:"storageClasses,omitempty"`
+	// VolumeSnapshotClasses contains the available VolumeSnapshotClasses in the Kubernetes cluster
+	// +optional
+	VolumeSnapshotClasses []VolumeSnapshotClass `json:"volumeSnapshotClasses,omitempty"`
+	// NodeInfos contains the information about the nodes in the Kubernetes cluster
+	// +optional
```

Comment:
why is nodeInfo optional?

Reply:
Because, it should not break, if the agent has a problem fetching nodes, e.g. because of permission problems.